main.py

```diff
@@ -101,6 +101,7 @@ def get_all_pipelines():
                 "id": pipeline_id,
                 "name": (pipeline.name if hasattr(pipeline, "name") else pipeline_id),
                 "valves": pipeline.valves if hasattr(pipeline, "valves") else None,
+                "rag_replacement": pipeline.rag_replacement if hasattr(pipeline, "rag_replacement") else False,
```

Comment:
Should be renamed to`file_handler`!

Reply:
Actually scratch that 😅, this method won't be compatible with OpenAI spec, and custom file handling for RAG pipeline should be done using native filter function.